### DISCUSSION TITLE
Installation of SCENIC+ development branch missing setup.py file?

### DISCUSSION THREAD

**Discussion by yls2g13** [aertslab/scenicplus#361] [Q&A]
Per the title. I cannot install the development branch as its missing a setup.py file. Am I missing something? 

Code used:
Source: https://github.com/aertslab/scenicplus/tree/development
```
conda create --name scenicplus python=3.11 -y 
conda activate scenicplus 
git clone https://github.com/aertslab/scenicplus 
cd scenicplus
git checkout development
pip install .
```

Error message:
```
$ pip install .
Directory '.' is not installable. File 'setup.py' not found.
```

**Comment by SeppeDeWinter** [comment]
Hi @yls2g13 

What a strange error..
What version of pip are you using?

Best,

Seppe

**Comment by yls2g13** [comment]
Ah this error is because of the wrong pip version - my bad. I'm now facing the same errors as seen in these posts, both related to `ModuleNotFoundError: No module named 'pyranges.pyranges_main'`:

https://github.com/aertslab/scenicplus/discussions/307
https://github.com/aertslab/scenicplus/issues/368

**Reply by SeppeDeWinter** [reply]
Hi @yls2g13 

Are you trying to load an old SCENIC+ object with a new environment?

All the best,

Seppe

**Reply by yls2g13** [reply]
Yes I am - does this mean I need to generate new SCENIC+ objects to match the new SCENIC+ envrionment?

**Reply by SeppeDeWinter** [reply]
Hi @yls2g13 

In your case I would suggest to continue using the old version of SCENIC+ (as you have already ran the pipeline), this you can find here https://github.com/aertslab/scenicplus/tree/old.

If you wish to re run SCENIC+ I would use the new environment.

Best,

Seppe
